packages/twenty-website/src/content/developers/self-hosting/setup.mdx

```diff
@@ -139,84 +139,86 @@ yarn command:prod cron:calendar:ongoing-stale
 yarn command:prod cron:workflow:automated-cron-trigger
 ```
 
-# Setup Environment Variables
+# Configuration Management
 
-## Frontend
+Twenty uses a hybrid approach for configuration management, combining environment variables with an admin panel interface.
 
-<ArticleTable options={[
-    ['REACT_APP_SERVER_BASE_URL', 'http://localhost:3000', 'Url of backend server'],
-    ['GENERATE_SOURCEMAP', 'false', 'Generate source maps for debugging'],
-    ['CHROMATIC_PROJECT_TOKEN', '', 'Chromatic token used for CI'],
-    ]}></ArticleTable>
+## Admin Panel Configuration
```

Comment:
Not super clear why this comes after. I'm told to set env vars for Google and then learn about this panel?